### PR TITLE
feat: Charge v1.1 (change position to risk)

### PR DESCRIPTION
**Pull request by RPDeshaies**
This PR changes the term `position (controlled, risky, desperate) to risk (low, moderate, high)`

### CODE REVIEW

File: public/documents/fari-rpgs/charge-rpg.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ Some actions may overlap with others. This is by design. As your campaign progre
 
 Make this game your own.
 
-As a player, you get to choose which action you roll by simply saying what your character does. Choosing one approach versus another might influence the position or effect of your action. The GM should tell you those details before you roll.
+As a player, you get to choose which action you roll by simply saying what your character does. Choosing one approach versus another might influence the risk or effect of your action. The GM should tell you those details before you roll.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:341]: Should this use formatting on "risk" and "effect"?

```suggestion
As a player, you get to choose which action you roll by simply saying what your character does. Choosing one approach versus another might influence the `risk` or `effect` of your action. The GM should tell you those details before you roll.
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 2]
```diff
@@ -475,31 +475,31 @@ The action rating is used to define the initial dice pool of the roll.
...
 > On a `6`, the PC wins and gets what they wanted. On a `4 or 5`, both the PC and the NPC have an effect on the result. On a > `1-3`, the NPC wins and the outcome results in a consequence for the PC.
 
-## 2. GM Sets the Position
+## 2. GM Sets the Risk
 
-The GM needs to set the `position` of the roll, based on the risks involved from failure.
+The GM sets the `risk` of the action based on what has been established in the narrative.
 
-A situation is either in a `controlled`, `risky`, or `desperate` position.
+The risk is either, low, moderate or high.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:482]: Adding formatting for keywords:

```suggestion
The risk is either, `low`, `moderate` or `high`.
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 3]
```diff
@@ -475,31 +475,31 @@ The action rating is used to define the initial dice pool of the roll.
...
+- When the risk is `high`, the odds are against you. Attempting to go against them might backfire heavily.
 
-By default, if you roll to overcome something, you are probably in a `risky` position. You wouldn’t be rolling if there was no risk involved.
+By default, an action roll has a `moderate` risk. You wouldn’t be rolling if there was no risk involved.
 
-The GM uses the narrative context to determine which position to pick.
+The GM uses the narrative context to determine which risk to pick.
 
-Another thing that might affect the position is the `action` being used. If a merchant you are trying to convince is in a bad mood, using `Sway` might be pretty `desperate`. `Bond` on the other hand could put you in a more `controlled` position.
+The action being used is another thing that might affect the risk of a roll — you are trying to convince a merchant who's in a bad mood, using `Sway` might be pretty `high risk`. `Bond` on the other hand could put you in a more `low risk` position, considering the situation.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:492]: Keyword formatting?

```suggestion
The action being used is another thing that might affect the `risk` of a roll — you are trying to convince a merchant who's in a bad mood, using `Sway` might be pretty `high risk`. `Bond` on the other hand could put you in a more `low risk` position, considering the situation.
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 4]
```diff
@@ -475,31 +475,31 @@ The action rating is used to define the initial dice pool of the roll.
...
 
-The position helps determine what failure looks like.
+The risk helps determine what failure looks like.
 
 ## 3. GM Sets the Effect Rating
 
-The position isn't the only thing that helps resolve the end result of a roll. Even in a desperate position, succeeding on a roll might still have a great `effect` on the narrative.
+The risk isn't the only thing that helps resolve the end result of a roll. Even in a high risk situation, succeeding on a roll might still have a great `effect` on the narrative.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:498]: Keyword formatting:

```suggestion
The risk isn't the only thing that helps resolve the end result of a roll. Even in a `high risk` situation, succeeding on a roll might still have a great `effect` on the narrative.
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 5]
```diff
@@ -475,31 +475,31 @@ The action rating is used to define the initial dice pool of the roll.
...
 
 In Charge, there are 3 possible effect ratings: `great`, `standard`, and `limited`. In most common scenarios, if you are rolling to overcome something, you are probably looking at generating a `standard` effect. The GM is responsible for telling you this information.
 
-If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but it's a more `controlled position`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a pretty `desperate` position, since it could create a lot of unwanted attention.
+If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but it's the `risk is low`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a `high risk` situation as it could create a lot of unwanted attention.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:502]: I'd reword this one a bit:

```suggestion
If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but the roll would be `low risk`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a `high risk` situation as it could create a lot of unwanted attention.
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 6]
```diff
@@ -653,13 +653,13 @@ This represents shifting circumstance. What you tried didn't work, and now there
 
 ### Worse Position
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:654]: Should this change to "increased risk" / Escalated Risk here?

```suggestion
### Escalated Risk
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 7]
```diff
@@ -713,7 +713,7 @@ Even with safety mechanics in place, accidents happen. That is why you should al
 >
 > - **Reduced Effect** : reduce the `effect` of the action by one level.
 > - **Complication**: tick one segment per `consequence rating` on a new or existing `danger clock`.
-> - **Worse Position** : reduce the `position` of the PC by one level for their next action rolls.
+> - **Worse position**: increase the `risk` of future related action rolls by one level.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:716]: Maybe needs to be "escalated risk"

```suggestion
> - **Escalated Risk**: increase the `risk` of future related action rolls by one level.
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 8]
```diff
@@ -1122,9 +1120,9 @@ We want character creation to be as streamlined as possible. Take the time to un
 
 Later in the campaign, when a milestone occurs, then you can think of new ways to make them truly unique using `talents`.
 
-> _Yann's character is a monster hunter. Consequently, he's pretty good at **Studying** and **Muscle** in general. Yann also just got a new **talent dot** and would like to excel at tracking monsters specifically. Because monsters aren't encountered that frequently in the game, he asks the GM if his talent could be "**+1 Position when I'm Shooting monsters**". The GM agrees. Now, Yann's character should be feared by most creatures roaming the lands._
+> _Yann's character is a monster hunter. Consequently, he's pretty good at **Studying** and **Muscle** in general. Yann also just got a new **talent dot** and would like to excel at tracking monsters specifically. Because monsters aren't encountered that frequently in the game, he asks the GM if his talent could be "**Recuded Risk when I'm Shooting monsters**". The GM agrees. Now, Yann's character should be feared by most creatures roaming the lands._
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:1123]: "Recuded" => "Reduced"

```suggestion
> _Yann's character is a monster hunter. Consequently, he's pretty good at **Studying** and **Muscle** in general. Yann also just got a new **talent dot** and would like to excel at tracking monsters specifically. Because monsters aren't encountered that frequently in the game, he asks the GM if his talent could be "**Reduced Risk when I'm Shooting monsters**". The GM agrees. Now, Yann's character should be feared by most creatures roaming the lands._
```

File: public/documents/fari-rpgs/charge-rpg.md [hunk 9]
```diff
@@ -1122,9 +1120,9 @@ We want character creation to be as streamlined as possible. Take the time to un
...
 >
-> _In another session, the GM realizes that Yann's talent is perhaps too generic, and could be tuned a little. He then asks if it could be rewritten as something like "**+1 Position when I'm Shooting monsters in the forest**" or something along those lines. Yann agrees, saying "My character lived in the forest all their life, and would probably be intimidated by big cities. I like it; let's go with this for now!"_
+> _In another session, the GM realizes that Yann's talent is perhaps too generic, and could be tuned a little. He then asks if it could be rewritten as something like "**Recuded Risk when I'm Shooting monsters in the forest**" or something along those lines. Yann agrees, saying "My character lived in the forest all their life, and would probably be intimidated by big cities. I like it; let's go with this for now!"_
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-rpg.md:1125]: "Recuded" => "Reduced"

```suggestion
> _In another session, the GM realizes that Yann's talent is perhaps too generic, and could be tuned a little. He then asks if it could be rewritten as something like "**Reduced Risk when I'm Shooting monsters in the forest**" or something along those lines. Yann agrees, saying "My character lived in the forest all their life, and would probably be intimidated by big cities. I like it; let's go with this for now!"_
```

File: public/documents/fari-rpgs/charge-srd.md [hunk 10]
```diff
@@ -349,15 +349,15 @@ The GM chooses one of the following:
 
 - `Reduced Effect` : reduce the `effect` of the action by one level.
 - `Complication`: tick one segment per `consequence rating` on a new or existing `danger clock`.
-- `Worse Position` : reduce the `position` of the PC by one level for their next action rolls.
+- `Worse position`: increase the _risk_ of future related action rolls by one level.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-srd.md:352]: Worse position => Escalated Risk?
```suggestion
- `Escalated Risk`: increase the _risk_ of future related action rolls by one level.
```

File: public/documents/fari-rpgs/charge-srd.md [hunk 11]
```diff
@@ -644,12 +644,12 @@ To do so, the GM looks at the 3 cards they've drawn, and builds a dice pool of `
...
+The result of this roll tells us what is going to be the initial `risk` the PCs will find themselves in when starting their next mission.
 
-- On a `1-3`, the PCs start in a `desperate` position.
-- On a `4-5`, the PCs start in a `risky` position.
-- On a `6`, the PCs start in a `controlled` position.
-- On a `66`, the PCs start in a `controlled` position, and gain some additional benefit.
+- On a `1-3`, the PCs start in a `high` risk position.
+- On a `4-5`, the PCs start in a `moderate` risk position.
+- On a `6`, the PCs start in a `low` risk position.
+- On a `66`, the PCs start in a `low` risk position, and gain some additional benefit.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/charge-srd.md:652]: Inconsistent formatting - sometimes `high` risk sometimes `high risk`

```suggestion
- On a `1-3`, the PCs start in a `high risk` position.
- On a `4-5`, the PCs start in a `moderate risk` position.
- On a `6`, the PCs start in a `low risk` position.
- On a `66`, the PCs start in a `low risk` position, and gain some additional benefit.
```

File: public/documents/fari-rpgs/dash.md [hunk 12]
```diff
@@ -64,10 +64,10 @@ _When momentum is spent for a roll, that roll cannot in turn generate momentum._
 If you make an action roll, and you get a partial success (4-5) or a bad outcome (1-3), you suffer a consequence.
 
 - **Reduced Effect** : reduce the _effect_ of the action by one level.
-- **Complication**: tick 1-3 segments on a new or existing _danger clock_, based on the _position_.
-- **Worse Position** : reduce the _position_ of the PC by one level for their next _action rolls_.
+- **Complication**: tick 1-3 segments on a new or existing _danger clock_, based on the _risk_.
+- **Worse position**: increase the _risk_ of future related action rolls by one level.
```
Comment (therabidbanana) [inline public/documents/fari-rpgs/dash.md:68]: Worse position => escalated risk?

```suggestion
- **Escalated Risk**: increase the _risk_ of future related action rolls by one level.
```

**Review by therabidbanana — APPROVED** [review]
I definitely think this is a change for the better - some notes on formatting and a couple typos.

There's also a few places where "Position" stays, I don't know if that's intentional.

File: public/documents/fari-rpgs/dash.md (hunk 12 above)
Comment (RPDeshaies) [reply public/documents/fari-rpgs/dash.md:68]: This is way better
